Tests/SentryTests/TestUtils/SentryTestObserver.m

```diff
@@ -95,8 +95,7 @@ - (void)testCase:(XCTestCase *)testCase didRecordIssue:(XCTIssue *)issue
 
 - (void)testBundleDidFinish:(NSBundle *)testBundle
```

Comment:
Where/how is this function called? I took a brief look at the code and grepping only turns up this line.

Reply:
Oh I see, it's an override.

Reply:
Would you mind adding `#pragma mark - XCTestObservation` above the `testCaseWillStart` override to make this more obvious? This is something I typically do for all protocol conformances to group them together in ObjC files and help show what the implementations are for.